### PR TITLE
Add book identifiers above the fold for librarians

### PR DESCRIPTION
**Pull request by jimchamp**
In order to aid librarians, the following changes have been made:
- Important identifiers are now available above the fold (for usergroup `librarians`).

### Screenshot

Screenshot from 2022-03-10 15-31-06

sticky_title

### CODE REVIEW

File: openlibrary/templates/type/edition/view.html [hunk 1]
```diff
@@ -249,16 +304,7 @@ <h4 class="year">&mdash;
...
-        $ seen = set()
-        $if previews:
-          Previews available in:
-          $for e in previews:
-            $if e.languages[0].name not in seen:
-              $ seen.add(e.languages[0].name)
-              <a href="$work.key?edition=$(e.ocaid)">$e.languages[0].name</a>
-      </p>
-
+      <a id="editions-list" name="editions-list" class="section-anchor"></a>
```
Comment (jimchamp) [inline openlibrary/templates/type/edition/view.html:307]:
```suggestion
```

**Comment by jimchamp** [comment]
Added omniline changes and removed sticky title.

Desktop view:
Screenshot from 2022-03-17 18-19-27

Mobile view:
Screenshot from 2022-03-17 18-19-12

File: static/css/components/work.less [hunk 2]
```diff
@@ -271,6 +284,20 @@ div.editionTools {
   }
 }
 
+@media only screen and (min-width: @width-breakpoint-tablet) {
+  .edition-omniline {
+    flex-direction: row;
+    justify-content: space-between;
+    overflow-x: scroll;
```
Comment (jimchamp) [inline static/css/components/work.less:291]:
```suggestion
    overflow-x: auto;
```

**Comment by mekarpeles** [comment]
lgtm ty!

**Pull request merged**